pulsar-client-api/src/main/java/org/apache/pulsar/client/api/ConsumerBuilder.java

```diff
@@ -687,7 +687,7 @@
      * the outstanding unchunked-messages by silently acking or asking broker to redeliver later by marking it unacked.
      * This behavior can be controlled by configuration: @autoAckOldestChunkedMessageOnQueueFull
      *
-     * @default 100
+     * The default value is 10.
```

Comment:
`@default xx` is the "fixed format" in Java API docs?

Reply:
`@default` is not the correct way to set the default value in the java doc. Using it will cause the default values to not show up in the java doc.